GreenhouseNode/GreenhouseNode/controllers/main.c

```diff
@@ -0,0 +1,18 @@
+/*
```

Comment:
This `main.c` should probably be deleted as it conflicts with our actual main file.